lib/json_schema/form_schemas.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# require_relative './json_schema/json_api_missing_attribute'
+
+module JsonSchema
+  class FormSchemas
+    def base_dir
```

Comment:
Is this too much future proofing?

Reply:
Its really just an abstraction out of what was in claims_api in modules/claims_api/lib/claims_api/form_schemas.rb which has been moved out so claims and appeals can share the code

Reply:
i love that appeals and clams are sharing this. very cool